test/index.js

```diff
@@ -51,13 +49,11 @@ describe('BIP39', function() {
 
   describe('generateMnemonic', function() {
     vectors.english.forEach(function(v, i) {
-      it('works for tests vector ' + i, sinon.test(function() {
-        this.mock(crypto).expects('randomBytes')
-          .exactly(1)
-          .onCall(0).returns(new Buffer(v[0], 'hex'))
+      it('works for tests vector ' + i, function() {
+        function rng() { return new Buffer(v[0], 'hex') }
 
-        assert.equal(BIP39.generateMnemonic(), v[1])
-      }))
+        assert.equal(BIP39.generateMnemonic(undefined, rng), v[1])
```

Comment:
The test you pointed me to is only testing the resulting word length. It has no assertion on which RNG we use as the default whatsoever. I think the default RNG is important enough for us to put some assertion around it. The original test at least says "we use the RNG returned by the `randombytes` module and we expect the module to do the right thing, whatever the right thing is defined in that library", in other words, the responsibility of using a cryptographically secure RNG is delegated to `randombytes`.